game.pro

```diff
@@ -33,10 +33,10 @@ unix:!macx {
 }
 
 win32{
-  INCLUDEPATH += C:/Qt/sfml/include
-  INCLUDEPATH += D:/Qt/sfml/include
   LIBS += -LC:/Qt/sfml/lib

```

Comment:
These lines must be put back in, else the rest of the cannot run the game anymore. I know I ignored it while we were fixing your setup and I think this is the moment to fix that :-)